### DISCUSSION TITLE
xit-mode for Emacs

### DISCUSSION THREAD

**Discussion by ryanolsonx** [jotaen/xit#25] [Show & tell]
I've started an Emacs package that we can collaborate on for Emacs [x]it! support. Right now, it supports:

- Checkboxes (the boxes themselves)
- Priority
- Tags (only simple #tagging currently)

But if you're an Emacs user and you're hoping to get basic xit support, this can be for you.

I'm hoping to work on this and the Vim tools. I'd love help though. If you have any desire to help, improve these code-bases. I'd gladly check and merge PRs.

@jotaen You can add Emacs: https://github.com/ryanolsonx/xit-mode/, if you'd like.

**Comment by jotaen** [comment]
🎉 That looks cool! I’ve put a link on the website.

I suppose the plugin/mode capabilities of Emacs are the most powerful ones, compared to other editors.

**Reply by ryanolsonx** [reply]
Thanks!

> I suppose the plugin/mode capabilities of Emacs are the most powerful ones, compared to other editors.

That's true in a lot of ways. For highlighting, it's regex based (not as powerful for some things). But for anything cool like checking or unchecking checkboxes, it can have a lot of power.
